tests/callbacks/test_model_checkpoint.py

```diff
@@ -0,0 +1,59 @@
+import pickle
```

Comment:
assume that this just move, no change here...

Reply:
correct, no new tests for checkpoints